distributed/deploy/spec.py

```diff
@@ -97,7 +97,8 @@ async def finished(self):
         await self._event_finished.wait()
 
     def __repr__(self):
-        return f"<{type(self).__name__}: status={self.status}>"
+        status = str(self.status).split(".")[1]
+        return f"<{type(self).__name__}: status={status}>"
```

Comment:
You can just use the enum name property.

```suggestion
        return f"<{type(self).__name__}: status={self.status.name}>"
```

Reply:
Oh! perfect. Yes, I have seen this used elsewhere as well in the code base.

Reply:
@freyam there is another one of these that needs updating too.

Reply:
My bad! I overlooked it. Fixed it now.